models/Element/Traits/DirtyIndicatorTrait.php

```diff
@@ -50,4 +50,9 @@ public function resetDirtyMap(): void
     {
         $this->dirtyFields = [];
     }
+
+    public function getDirtyFields(): array
+    {
+        return $this->dirtyFields;
```

Comment:
I think it makes more sense to get the fields as value, right?
```suggestion
    /**
     * @return string[]
     */
    public function getDirtyFields(): array
    {
        return array_keys($this->dirtyFields);
```

Reply:
True, thank you for the suggestion